### DISCUSSION TITLE
R 5.2.0 Sunbird Inquiry Building Block Regression Phase Sign Off

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-inQuiry/Community#44] [General Discussions]
Hello All,

This is Regression Phase QA Sign off for the release 5.2.0 for Sunbird Inquiry Building Block.

Environment: Sunbird Inquiry Portal
Testing Status: Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)
Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=706217685)

cc: @pallakartheekreddy @Nadiya496 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran

**Comment by Nadiya496** [comment]
Hello All,

As part of 5.2 Inquiry building block, we have made a CSP related changes, deployed validated the same with all CRUD and migrated data as well. It is been done on the  Sunbird Inquiry Portal itself. Please find the details for the same.
CSP Related tickets: [IQ-149](https://project-sunbird.atlassian.net/browse/IQ-149) and [IQ-193](https://project-sunbird.atlassian.net/browse/IQ-193). Both have been updated in the sign off template in the regression phase though it came as hotfix.

Environment: Sunbird Inquiry Portal ([Link](https://dev.inquiry.sunbird.org/))
Testing Status: Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10609)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=0)
Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1Vbw9bIMO_bGs4GawSqTkKfvuzDuo7R9wzQRWtBE359M/edit#gid=706217685)


Note: CSP related changes , steps, procedure will be updated in the microsite by today EOD.

cc: cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @luckynvdu @rajeevsathish @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @PoojaJManjooran
